pkg/proxy/iptables/proxier.go

```diff
@@ -190,14 +189,6 @@ func NewProxier(ipt utiliptables.Interface, exec utilexec.Interface, syncPeriod
 		return nil, fmt.Errorf("can't set sysctl %s: %v", sysctlRouteLocalnet, err)
 	}
 
-	// Load the module.  It's OK if this fails (e.g. the module is not present)
-	// because we'll catch the error on the sysctl, which is what we actually
-	// care about.
-	exec.Command("modprobe", "br-netfilter").CombinedOutput()
```

Comment:
> Can you log something here in case someone already has a plugin that needs br-netfilter and they're getting it for free?

Ok, but not quite sure what you're asking for...  logging that the load failed?  it's expected to fail in a bunch of systems because br-netfilter used to be built-in to the bridge module, but got split sometime in the late 3.1x timeframe.  So new systems need this, older ones don't.